graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -20,6 +20,41 @@
 )
 
 
+def _get_inherited_field_types(class_to_field_type_overrides, schema_graph):
```

Comment:
Move code below into these three next functions. Also instead of having hidden classes as a parameter, I just hide classes with no properties. Although it removes the sanity check that all classes have properties, it simplifies the user interface. Furthermore, I am going to add this sanity check back in defacto later by asserting that all classes are in the schema except for Vertex and Trait which have no fields.